django_ses/management/commands/get_ses_statistics.py

```diff
@@ -32,6 +32,8 @@ def handle(self, *args, **options):
         connection = SESConnection(
             aws_access_key_id=settings.ACCESS_KEY,
             aws_secret_access_key=settings.SECRET_KEY,
+            proxy=settings.AWS_SES_PROXY,
```

Comment:
You can test this setting is picked up like so:
https://github.com/django-ses/django-ses/blob/master/django_ses/tests/test_settings.py

And that it is passed to the `SESConnection` by modifying this method to store all the args and kwargs on the fake instance, then testing they are there:
https://github.com/django-ses/django-ses/blob/master/django_ses/tests/test_backend.py#L28